LICENSE

```diff
@@ -1,7 +1,3 @@
-All files of this project under the directory "from_3b1b" are copyright 3Blue1Brown LLC and used by permission for this project only.
-
-Any other file of this project is available under the MIT license as follow:
-
 MIT License
 
 Copyright (c) 2018 3Blue1Brown LLC
```

Comment:
This should be changed. It's probably supposed to be replaced with something else in the normal MIT license.

Reply:
Woops good catch. [Apparently it is customary](https://en.wikipedia.org/wiki/MIT_License) to add the name of the copyright holder at the top of the MIT License, even though it is very lax. What name should we use? The "manim community developers"? Or leave it blank?

Reply:
Numpy did this
https://github.com/numpy/numpy/blob/master/LICENSE.txt#L1

I guess Manim Community Developers is fine

Reply:
Done. I'll request others to look at it to confirm a quorum.